### PR TITLE
[release + cherry pick] run verify codesign test on post submit only

### PR DESCRIPTION
**Pull request by XilaiZhang**
cherry picks https://github.com/flutter/flutter/pull/120411

### CODE REVIEW

File: .ci.yaml [hunk 1]
```diff
@@ -3114,6 +3114,7 @@ targets:
     enabled_branches:
       - flutter-\d+\.\d+-candidate\.\d+
     recipe: flutter/flutter
+    presubmit: false
```
Comment (CaseyHillers) [inline .ci.yaml:3117]: Can you file a CP request?

Comment (XilaiZhang) [reply .ci.yaml:3117]: Thanks for the reminder! Forgot that there is a process for cp now.

Comment (XilaiZhang) [reply .ci.yaml:3117]: https://github.com/flutter/flutter/issues/120466 for cp request

**Comment by XilaiZhang** [comment]
Umm it seems like this one wasn't merged to stable, and that might be the reason why we are still seeing presubmit checks on stable?

**Comment by XilaiZhang** [comment]
Closing this since it looks like 3.7.1 is no longer the stable anymore

**Pull request closed without merging**